ksql-engine/src/main/java/io/confluent/ksql/planner/plan/AggregateNode.java

```diff
@@ -407,17 +402,19 @@ private void collectAggregateArgExpressions(
         final Collection<? extends Expression> expressions,
         final Set<String> seen
     ) {
-      expressions.stream()
-          .filter(e -> !seen.contains(e.toString()))
-          .forEach(expression -> {
-            final String internalColumnName = INTERNAL_COLUMN_NAME_PREFIX
-                + aggArgExpansionList.size();
-            seen.add(expression.toString());
-            internalNameToIndexMap.put(internalColumnName, aggArgExpansionList.size());
-            aggArgExpansionList.add(SelectExpression.of(internalColumnName, expression));
-            expressionToInternalColumnNameMap
-                .putIfAbsent(expression.toString(), internalColumnName);
-          });
+      for (final Expression expression : expressions) {
```

Comment:
Note, I've moved away from the streams impl as its potentially incorrect as the `filter` call could do the wrong thing if all elements are filtered before any are passed to `forEach`, which the stream impl is free to do if it wants.